src/components/Item/index.js

```diff
@@ -0,0 +1,10 @@
+import { connect } from 'react-redux';
```

Comment:
Please **don't connect to Redux** yet. We should discuss how our **data model** will look like first. Probably it will be something like (initialState for cart):
```
cart: {
  items: [] // your items are hre
}
```